src/content/en/updates/2018/06/webar-chacmool.md

```diff
@@ -0,0 +1,121 @@
+experiences. The web makes AR convenient and accessible everywhere.
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="Zu6MXyfi-Ts"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+We have now enabled this demo on
+[Chrome Canary](https://play.google.com/store/apps/details?id=com.chrome.canary)
+on
+[ARCore-compatible Android devices](/ar/discover/supported-devices) with Android O or later.
+ This work relies on a new WebXR proposal (the
+[WebXR Hit Test API](https://github.com/immersive-web/hit-test/blob/master/explainer.md)),
+so it is under a flag and intended to stay in Canary as we test and refine the
+new API proposal with other members of the Immersive Web Community Group. In
+fact, to access the demo you'll need to enable two flags in `chrome://flags:`
+`#webxr` and `#webxr-hit-test`.  Once you have these both enabled and have restarted
+Canary, you can check out the [Chacmool](https://web-education-ar-demo.appspot.com/) demo.  
+
+The [Chacmool AR experience](https://web-education-ar-demo.appspot.com/) is centered around
+education, leveraging AR's immersive and interactive nature to help users learn
+about ancient Chacmool sculptures. Users can place a life size statue in their
+reality and move around to see the sculpture from various different angles and
+distances. The immersive nature of AR allows users to freely explore, discover
+and play with content, just like they can in the real world. When viewing an
+object in AR, as opposed to seeing it on a flat 2D screen, we are able to get a
+deep understanding of what we are looking at because we can see it from many
+different angles and distances using a very intuitive interaction model: walking
+around the object, and getting physically closer or further away.   Also, in
+this experience, there are annotations placed directly on the sculpture. This
+enables users to directly connect what is described in text and where those
+features are on the sculpture.
+
+This demo was built over the course of about a month to build, leveraging some
+of the components from the WebXR team's first web based AR demo,
+[WebAR-Article](https://blog.google/products/google-vr/augmented-reality-web-everyone/).
+Information about the sculpture was sourced from its
+[Google's Arts & Culture](https://artsandculture.google.com/asset/a-chacmool-figure-holding-a-bowl-in-the-ruins-of-templo-mayor-in-mexico-city/nQFDnZcHsxuiiA)
+page, and the 3D model was provided by Google Arts & Culture's partner,
+[CyArk](https://artsandculture.google.com/partner/cyark).  To get the 3D model
+ready for the web, a combination of Meshlab, and Mesh Mixer was used to repair
+the model and decimate its mesh to decrease its file size. Then
+[Draco, a library for compressing and decompressing 3D geometric meshes and point clouds](https://github.com/google/draco)
```

Comment:
Should be: `[Draco](https://github.com/google/draco), a library for compressing and decompressing 3D geometric meshes and point clouds`